extensions/amp-video-iframe/1.0/amp-video-iframe.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {BUBBLE_MESSAGE_EVENTS} from '../0.1/amp-video-iframe';
+import {VideoBaseElement} from '../../amp-video/1.0/base-element';
+import {VideoIframe} from '../../amp-video/1.0/video-iframe';
+import {createCustomEvent} from '../../../src/event-helper';
+import {dict} from '../../../src/utils/object';
+
+/** @const {string} */
+const TAG = 'amp-video-iframe';
+
+class AmpVideoIframe extends VideoBaseElement {}
+
+/**
+ * @param {!MessageEvent} e
+ */
+const onMessage = (e) => {
+  const method = e.data?.method;
+  if (method) {
+    if (method == 'getIntersection') {
+      // TODO
+      return;
+    }
+    throw new Error(`Unknown method ${method}`);
+  }
+  const event = e.data?.event;
+  if (!event) {
+    return;
+  }
+  if (event === 'analytics') {
+    // TODO
+    return;
+  }
```

Comment:
In Classic AMP, this dispatches an event:

https://github.com/ampproject/amphtml/blob/1bfb403c483515c9bdb141294c809b7362583a04/extensions/amp-video-iframe/0.1/amp-video-iframe.js#L323-L327

Not sure of how we're handling analytics for Bento components. Probably trigger the same kind of event too?

Reply:
Why not just trigger the exact same event? This is the AMP binding code so we can do whatever AMP needs here.